### PR TITLE
- updated readme

### PR DESCRIPTION
**Pull request by SencyCaspit**

### CODE REVIEW

File: Assessment.md [hunk 1]
```diff
@@ -27,20 +27,25 @@ class MainActivity : ComponentActivity(), SMKitUIWorkoutListener {
 fun startAssessment() {
     try {
         // In order to start Sency's blueprint assessment
-        smKitUI.startAssessment(listener = this)
+        smKitUI.startAssessment(
+            Fitness, // Assessment Type
+            listener = this, // Assessment Listener
+            userData = UserData(age = 28, gender = Gender.Female) // User Data
```
Comment (ofergoldstein) [inline Assessment.md:33]: @SencyCaspit we should mention that it's optional

File: Assessment.md [hunk 2]
```diff
@@ -27,20 +27,25 @@ class MainActivity : ComponentActivity(), SMKitUIWorkoutListener {
...
+Passing User Data programmtically is optional, not passing will display User Details Page 
 
 ### Start Custom Assessment
 **startAssessment** and **Custom** starts one of your own Custom Assessments.
 ```Kotlin
 fun startAssessment() {
     try {
         // In order to start your own custom assessment
-        smKitUI.startAssessment(Custom("YOUR_CUSTOM_ID"), listener = this)
+        smKitUI.startAssessment(Custom("YOUR_CUSTOM_ID"), listener = this, userData = null)
```
Comment (ofergoldstein) [inline Assessment.md:48]: @SencyCaspit you can reformat this line such that it's similar to how it's formatted in the example above

File: Assessment.md (hunk 1 above)
Comment (SencyCaspit) [reply Assessment.md:33]: @ofergoldstein i mentioned it below, you want me to mention it also up here?

File: Assessment.md (hunk 2 above)
Comment (SencyCaspit) [reply Assessment.md:48]: @ofergoldstein fixing it

File: Assessment.md (hunk 1 above)
Comment (ofergoldstein) [reply Assessment.md:33]: @SencyCaspit yes
`userData = UserData(age = 28, gender = Gender.Female) // User Data (optional) `

**Review by ofergoldstein — APPROVED** [review]

**Pull request merged**